public/API/API_GetGameHashes.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ *  API_GetGameHashes - returns information about supported game files for a specific game
+ *    i : game id
+ *
+ *  array
+ *   object     [value]
+ *    string     Name        name given to the hash, usually a filename
+ *    string     MD5         unique hash
+ *    array      Labels
+ *     string     [value]    labels associated to the hash such as "nointro" or "redump"
+ *    string     PatchUrl    link to RAPatches URL for the hash
+ */
+
+use App\Models\Game;
+use App\Platform\Data\GameHashData;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Validator;
+
+$input = Validator::validate(Arr::wrap(request()->query()), [
+    'i' => ['required', 'min:1'],
+]);
+
+$game = Game::find($input['i']);
+
+if (!$game) {
+    return response()->json([], 404);
```

Comment:
Should the 404 response be an object instead of an array, perhaps with an empty "Results" to match a game with no hashes?

Game with no hashes:
```
{"Results":[]}
```

Invalid game ID:
```
[]
```

Reply:
No objection here - updated in latest.